packages/react-instantsearch-core/src/hooks/__tests__/useInstantSearch.test.tsx

```diff
@@ -24,8 +23,8 @@ describe('useInstantSearch', () => {
     test('it errors when not nested in InstantSearch', () => {
       const { result } = renderHook(() => useInstantSearch());
```

Comment:
seems like this error now throws globally, can use toThrowError instead of result.current.error I guess